features/ui/sidebar-navigation/sidebar-navigation.tsx

```diff
@@ -193,7 +193,11 @@ export function SidebarNavigation() {
               text="Support"
               iconSrc="/icons/support.svg"
               isCollapsed={isSidebarCollapsed}
-              onClick={() => alert("Support")}
+              onClick={() =>
```

Comment:
You can also turn this into a `MenuItemLink` and use `href="mailto:support@prolog-app.com?subject=Support Request:" instead. That makes it also easier to test this